libraries/AC_PrecLand/AC_PrecLand.cpp

```diff
@@ -89,7 +89,17 @@ const AP_Param::GroupInfo AC_PrecLand::var_info[] = {
     // @Values: -1:DefaultBus,0:InternalI2C,1:ExternalI2C
     // @User: Advanced
     AP_GROUPINFO("BUS",    8, AC_PrecLand, _bus, -1),
-    
+
+    // @Param: LAG
+    // @DisplayName: Precision Landing sensor lag
+    // @Description: Precision Landing sensor lag in ms, to cope with variable landing_target latency
+    // @Range: 1 250
+    // @Increment: 1
+    // @Units: ms
```

Comment:
Any chance of making this a float instead? I thought that was the usual target for new user facing parameters for the most part?

Reply:
I guess you mean have the LAG defined in seconds instead of milliseconds?  I don't think we need to use a float if using milliseconds..

Reply:
Yeah. If we want milliseconds I'm not going to oppose it, but it would be nice to have the name be `LAG_MS` then.